gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 606
+cordaApiRevision = 607
```

Comment:
This change is appearing to me as `606` -> `607`. Is that the same for you? I'm questioning it because Yiftach just merged and this is `607` in the release branch now so the diff seems wrong..

Reply:
yeah, I think I need to do a merge with mainline and bump again. Will do one now.

Reply:
Cool, thanks. I thought the diff would have been done against the release branch originally and should have looked different in the diff. Looks good now anyway.